unit-tests/live/d500/test-hw-timestamp.py

```diff
@@ -0,0 +1,57 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+# test:donotrun
```

Comment:
Should enable the test

Reply:
This test cannot be enabled right now, because we have no device that supports it in our CI, and this would fail it

Reply:
OK, then maybe add a comment explaining why the test is disabled and when it should be enabled